test/python/quantum_info/operators/symplectic/test_sparse_pauli_op.py

```diff
@@ -510,6 +510,132 @@ def test_simplify_zero(self, num_qubits):
         np.testing.assert_array_equal(zero_op.paulis.phase, np.zeros(zero_op.size))
         np.testing.assert_array_equal(simplified_op.paulis.phase, np.zeros(simplified_op.size))
 
+    def test_sort(self):
+        """Test sort method."""
+        with self.assertRaises(QiskitError):
+            target = SparsePauliOp([], [])
```

Comment:
Shouldn't sorting on an empty set work? I would argue since it's empty it's already "sorted" anyways 😄